src/libstd/sys_common/backtrace.rs

```diff
@@ -100,16 +103,25 @@ fn filter_frames(frames: &[Frame],
         return (0, 0);
     }
 
-    let skipped_before = 0;
+    // Look for the first occurence of a panic entry point
+    // Skip all frames before that
+    let skipped_before = frames.iter().position(|frame| {
```

Comment:
Did these leak in by accident? I think tidy failed to compile due to these maybe?

Reply:
I forgot to commit my code...